tests/unit_tests/models/test_bravo.py

```diff
@@ -176,7 +172,7 @@ def test_get_prediction_vector_for_time(tmpdir, data_processing_configs):
     window_width_target = pd.Timedelta(
         data_processing_configs["input_output_window"]["window_width_target"]
     )
-    bin_interval = pd.Timedelta(data_processing_configs["resample"]["bin_interval"])
+    bin_interval = pd.Timedelta(configs["data_processing"]["resample"]["bin_interval"])
```

Comment:
@haneslinger is this intentional? not using `data_processing_configs`

Reply:
it is intentional. `data_processing_configs` no longer has bin interval.